apps/coordinator/ts/deployer/deployer.controller.ts

```diff
@@ -34,11 +35,12 @@ export class DeployerController {
   @ApiResponse({ status: HttpStatus.CREATED, description: "The MACI contracts were successfully deployed" })
   @ApiResponse({ status: HttpStatus.FORBIDDEN, description: "Forbidden" })
   @ApiResponse({ status: HttpStatus.BAD_REQUEST, description: "BadRequest" })
+  @ApiResponse({ status: HttpStatus.INTERNAL_SERVER_ERROR, description: "InternalServerError" })
```

Comment:
Could we add more explicitly messages to the Swagger decorators? Expand it more than "InternalServerError"